code/modules/projectiles/guncrafting/handle.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/weapon/gun_attachment/handle
+	name = "base"
+	desc = "sfdahgfhgasfhgiahjfhajsdjkJA"
```

Comment:
no, have a suitable base description.
like, "A handle! but not one for doors" (idk shutup)